lib/game.rb

```diff
@@ -0,0 +1,81 @@
+class Game
+    include Interface
+
+    def initialize(player1, player2, board)
+        @player1 = player1
+        @player2 = player2
+        @board = board
+        @status = "initial"
+    end
+
+    def initial
```

Comment:
Like my comments in both `board` and `player.rb`, perhaps this method is unnecessary/redundant?

Reply:
same answer above.